tests/network/vmi_networking.go

```diff
@@ -819,7 +819,7 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				Entry("with custom CIDR [IPv6]", []v1.Port{}, 8080, "fd10:10:10::/120"),
 			)
 
-			It("[outside_connectivity]should be able to reach the outside world [IPv6]", func() {
+			It("[outside_connectivity][ARM64InfraNotSupported]should be able to reach the outside world [IPv6]", func() {
```

Comment:
Isn't this not-supported because it runs on `kind`?
But I also see it is being skipped for conformance tests [1] and there is no apparent reason to add another tag/flag.

[1] https://github.com/kubevirt/kubevirt/blob/5b2b9f4243470b8a7376b5f64b9c7701a2efc630/hack/conformance.sh#L29

Reply:
Answered in https://github.com/kubevirt/kubevirt/pull/10303/files#r1361436064